src/language/visitor.ts

```diff
@@ -76,9 +76,14 @@ type ReducedField<T, R> = T extends null | undefined
   ? ReadonlyArray<R>
   : R;
 
-const QueryDocumentKeys = {
-  Name: [],
+/**
+ * A KeyMap describes each the traversable properties of each kind of node.
+ */
+export type VisitorKeyMap<KindToNode> = {
```

Comment:
Since we dropped support for custom ASTs we don't need `KindToNode` argument also I renamed `VisitorKeyMap` to `ASTVisitorKeyMap` and will provide an alias for `v15`.